### PR TITLE
Fill in various "?" cells from the element and attribute tables

### LINKED ISSUE #97: AxAPI completion
In both the role, and the state and propertey tables, there are a number of ? placeholders in the AxAPI columns. These need to be updated to include actual data.

### PR DESCRIPTION
**Pull request by scottaohara**
Closes #97

The following tasks have been completed:
 * [x] Confirmed there are no ReSpec/BikeShed errors or warnings.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/html-aam/pull/245.html" title="Last updated on Dec 10, 2019, 1:09 AM UTC (4cc0a0f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/html-aam/245/45389a7...4cc0a0f.html" title="Last updated on Dec 10, 2019, 1:09 AM UTC (4cc0a0f)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3455,7 +3455,7 @@ <h3>HTML Element Role Mappings</h3>
                     "text-underline-style:solid" text attribute on its text container.
                   </div>
                 </td>
-                <td class="ax">?</td>
+                <td class="ax">Not mapped</td>
```
Comment (scottaohara) [inline index.html:3509]: this will be updated further in #226

**Comment by scottaohara** [comment]
outside of ARIA mappings (and even some of those were taken care of in #244 and future work in #259) the attributes table no longer has missing mapping entries.

This PR additionally revises some of the existing mappings to point to Core AAM.

File: index.html [hunk 2]
```diff
@@ -5166,75 +5260,85 @@ <h3>HTML Attribute State and Property Mappings</h3>
...
+                <a data-cite="html/scripting.html#attr-script-src">`script`</a>;
+                <a data-cite="html/embedded-content.html#attr-source-src">`source`</a>;
+                <a data-cite="html/media.html#attr-track-src">`track`</a>;
+                <a data-cite="html/media.html#attr-media-src">`video`</a>
+              </td>
+              <td class="aria"><div class="general">Not mapped</div></td>
+              <td class="ia2">
+                <div class="objattrs">
+                  <span class="type">Object attributes:</span>
+                  `src` on <a href="#el-img">`img`</a> only
```
Comment (scottaohara) [inline index.html:5381]: see #266

**Review by stevefaulkner — APPROVED** [review]

**Comment by scottaohara** [comment]
@marcoscaceres can you tell us why this build is failing?

**Comment by marcoscaceres** [comment]
Having a look...

**Comment by marcoscaceres** [comment]
Some of the scripts are depending on things that we no longer ship with in ReSpec (e.g., RequireJS). I've got a rough patch locally that I'll send up soon that should fix things.

**Comment by scottaohara** [comment]
@marcoscaceres how'd the patch go?  is this safe to merge in regardless of the fail?

**Pull request merged**